.eslintrc.json

```diff
@@ -1,30 +1,28 @@
 {
-    "env": {
-        "browser": true,
-        "es6": true,
-        "jest" : true
-    },
-    "extends": [
-        "airbnb-base"
-    ],
-    "globals": {
-        "Atomics": "readonly",
-        "SharedArrayBuffer": "readonly"
-    },
-    "parser": "@typescript-eslint/parser",
-    "parserOptions": {
-        "ecmaVersion": 2018,
-        "sourceType": "module"
-    },
-    "plugins": [
-        "@typescript-eslint",
-        "jest"
-    ],
-    "rules": {
-        "jest/no-disabled-tests": "warn",
-        "jest/no-focused-tests": "error",
-        "jest/no-identical-title": "error",
-        "jest/prefer-to-have-length": "warn",
-        "jest/valid-expect": "error"
-    }
-}
\ No newline at end of file
+  "env": {
+    "browser": true,
+    "es6": true,
```

Comment:
Silly JavaScript question: what does this setting mean if the project uses Typescript?

Reply:
Follow-up question: if we'd like to make this library generally available, are we building it in a way that can be used cross-browser? (e.g. without ES6 polyfills.)

Reply:
This is just the linter config but you have a point about older browsers, I've set the target to ES5 in tsconfig.json so this should put in the ES6 polyfils we need.

Reply:
Cheers. 👍 

What does it mean to lint the code as ES6, when the code is written in TypeScript rather than JavaScript? Is there such a thing as ES5/ES6 TypeScript? Sorry if that's a very confused question. 🙂

Reply:
There's probably no such thing as ES5 Typescript but eslint will lint javascript as well as Typescript which is why the setting exists. 
As to why it's here, it's what came out of the defaults when I ran the setup script.